### PR TITLE
Remove forwarding functions in Window.cpp

### LINKED ISSUE #21505: Remove forwarding functions in Window.cpp
https://github.com/OpenRCT2/OpenRCT2/blob/412ca5d9c54b90e1d9468e7c4f6bfe27d5577d92/src/openrct2/interface/Window.cpp#L1398-L1528

These functions should be removed and replaced with direct calls to the window class functions.

As mentioned here https://github.com/OpenRCT2/OpenRCT2/pull/21460#discussion_r1502534987

### PR DESCRIPTION
**Pull request by mrmbernardi**
Resolves #21505.

I'd like another set of eyes to go over this and make sure I haven't accidentally turned MouseUp into MouseDown somewhere or something like that.

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]
Looks correct. Have you tested the text input stuff a bit as there might be some change in operation due to the changes.

File: src/openrct2/interface/Window.cpp [hunk 1]
```diff
@@ -894,7 +895,7 @@ void WindowScrollToLocation(WindowBase& w, const CoordsXYZ& coords)
  */
 static void call_event_viewport_rotate_on_all_windows()
```
Comment (mrmbernardi) [inline src/openrct2/interface/Window.cpp:896]: There's a snake in my .cpp

**Comment by mrmbernardi** [comment]
> Looks correct. Have you tested the text input stuff a bit as there might be some change in operation due to the changes.

I ran the code paths and didn't see anything odd. The code is relevant when you close/cancel/complete a dialogue box e.g. the one you see when renaming the park or when you unfocus a text box in a window e.g. the scenery search field.

**Comment by Gymnasiast** [comment]
Mind the milestone.

**Pull request merged**